### PR TITLE
Add OrmFactory tests

### PR DESCRIPTION
**Pull request by roxblnfk**
| Q             | A
| ------------- | ---
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | #22

### CODE REVIEW

File: src/Exception/SchemaNotReadException.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Cycle\Exception;
+
+use RuntimeException;
+use Yiisoft\FriendlyException\FriendlyExceptionInterface;
+
+class SchemaNotReadException extends RuntimeException implements FriendlyExceptionInterface
```
Comment (vjik) [inline src/Exception/SchemaNotReadException.php:10]: Maybe extends [Cycle\ORM\Exception\SchemaException](https://github.com/cycle/orm/blob/master/src/Exception/SchemaException.php)?

Comment (roxblnfk) [reply src/Exception/SchemaNotReadException.php:10]: Or [FactoryException](https://github.com/cycle/orm/blob/master/src/Exception/FactoryException.php)

Comment (samdark) [reply src/Exception/SchemaNotReadException.php:10]: `CanNotReadSchemaException`?

Comment (samdark) [reply src/Exception/SchemaNotReadException.php:10]: `SchemaReadingException`?

File: src/Exception/SchemaNotReadException.php [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+{
+    public function __construct()
+    {
+        parent::__construct('Cycle Schema not read.');
```
Comment (samdark) [inline src/Exception/SchemaNotReadException.php:14]: Am I correct about the meaning?

```suggestion
        parent::__construct('Cycle Schema can not be read.');
```

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
Looks alright now. If you don't have anything to add it, feel free to merge it,

**Pull request merged**